hooks/useModal.js

```diff
@@ -0,0 +1,16 @@
+import { useState } from 'react';
+
+const useModal = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggle = () => {
+    setIsOpen(!isOpen);
+  }
+
+  return {
```

Comment:
It's more reutilizable If We just return an Array, because we can rename It as We pleased